pkg/model/model.go

```diff
@@ -296,8 +344,61 @@ func (m *Model) GetCRDs() ([]*CRD, error) {
 		// Now add the additional printer columns that have been defined explicitly
 		// in additional_columns
 		crd.addAdditionalPrinterColumns(m.cfg.GetAdditionalColumns(crdName))
-
 		crds = append(crds, crd)
+
+		for _, toInjectNestedFields := range alltoInjectNestedFields {
+
+			addThis := ""
+			var previous *Field
+			var res bool
+
+			len := len(toInjectNestedFields.arr)
```

Comment:
`len` is a Go keyword, i suggest just to use `len(toInjectNestedFields.arr)` whenever you need to.